chipsec/chipset.py

```diff
@@ -162,6 +162,9 @@ class RegisterType:
 # Xeon v3 Processor (Haswell Server)
 0x2F00 : {'name' : 'Haswell Server', 'id' : CHIPSET_ID_HSX,  'code' : CHIPSET_CODE_HSX,  'longname' : 'Server 4th Generation Core Processor (Haswell Server CPU / Wellsburg PCH)'},
 
+# Xeon v5 Processor (Skylake Server)
+0x1918 : {'name' : 'Skylake Server', 'id' : CHIPSET_ID_SKL,  'code' : CHIPSET_CODE_SKL,  'longname' : 'Server 6th Generation Core Processor (Skylake CPU / Sunrise Point PCH)'},
```

Comment:
Thanks! Can you please use "Intel Xeon Processor E3 v5 (Skylake CPU / Sunrise Point PCH)" as the longname?